packages/ozone/src/mod-service/index.ts

```diff
@@ -341,6 +357,15 @@ export class ModerationService {
       .returningAll()
       .executeTakeFirstOrThrow()
 
+    // If reporting is muted for this reporter, we don't want to create a subject status
+    if (meta.isReporterMuted) {
+      const subjectStatus = await this.getStatus(subject)
+      return {
+        event: modEvent,
+        subjectStatus,
+      }
+    }
```

Comment:
Could we handle this logic in `adjustModerationSubjectStatus()`?  It seems to have all the necessary information, and might just be a little more uniform to do it there.